components/automate-ui/src/app/entities/service-groups/service-groups.requests.ts

```diff
@@ -151,7 +151,7 @@ export class ServiceGroupsRequests {
     }, searchParam, filters);
   }
 
-  public deleteServicesById(listOfIds: number[]): any {
+  public deleteServicesById(listOfIds: number[]): Observable<any> {
```

Comment:
Thanks to @msorens for helping realize the error in this related effect was coming from the lack of an Observable signature.  The <any> should be updated to be more specific as well

Reply:
Looking into this more, I'm going to investigate the proper type in a new branch so as not to hold up these dependencies.